torch/csrc/jit/passes/remove_inplace_ops.cpp

```diff
@@ -42,9 +48,21 @@ void RemoveInplaceOps(Block* block) {
         newNode->addInput(input);
       }
 
+      size_t additionalInputCount = 0;
+      if (additionalNoneArgumentCount.find(node->kind()) != additionalNoneArgumentCount.end()) {
+        additionalInputCount = additionalNoneArgumentCount.at(node->kind());
+      }
+
+      for (size_t i = 0; i < additionalInputCount; ++i) {
```

Comment:
Why do we unconditionally insert additionalInputCount None node? Don't we need to check whether the node already has enough inputs?

Reply:
Thanks, I'll fix this

Reply:
Thanks, I'll fix this

Reply:
Actually, this is because both `zero_` and `fill_` have fixed number of inputs, so the number of lacked inputs is fixed. But i'll update the code to be more clear.